sdfObject/sdfobject-ipso-temperature-sdfchoice.sdf.json

```diff
@@ -0,0 +1,149 @@
+{
+  "info": {
+    "title": "OMA LwM2M Temperature (Object ID 3303)",
+    "version": "2023-03-25",
+    "copyright": "Copyright 2019 Open Mobile Alliance.",
+    "license": "BSD-3-Clause"
+  },
+  "namespace": {
+    "oma": "https://onedm.org/ecosystem/oma"
+  },
+  "defaultNamespace": "oma",
+  "sdfObject": {
+    "Temperature": {
+      "label": "Temperature",
+      "description": "This IPSO object should be used with a temperature sensor to report a temperature measurement.  It also provides resources for minimum/maximum measured values and the minimum/maximum range that can be measured by the temperature sensor. An example measurement unit is degrees Celsius.",
+      "oma:id": "3303",
+      "sdfProperty": {
+        "Sensor_Value": {
+          "label": "Sensor Value",
+          "description": "Last or Current Measured Value from the Sensor.",
+          "oma:id": 5700,
```

Comment:
It is probably just me, but I can't find mentions of such namespace-specific qualities in draft-13.

Reply:
Usually in SDF models you would leave ecosystem specific details (like OMA IDs) to the mapping file, but I happened to run the conversion tool with ID in-lining mode in this case.